src/plots/cartesian/layout_attributes.js

```diff
@@ -180,7 +180,7 @@ module.exports = {
         description: [
             'Sets the text displayed at the ticks position via `tickvals`.',
             'Only has an effect if `tickmode` is set to *array*.',
```

Comment:
ahh yes you're totally right, I wasn't thinking about opt 2. 
We should probably leave as is for now then. (but just change the typo in this commit)